solidity/ecdsa/hardhat.config.ts

```diff
@@ -46,6 +46,15 @@ export const testConfig = {
 const config: HardhatUserConfig = {
   solidity: {
     compilers: [
+      {
+        version: "0.8.17",
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 200,
+          },
+        },
+      },
```

Comment:
Shouldn't we remove the config for `0.8.9`?

Reply:
We can't. We still need 0.8.9 compiler to compile contracts from `@threshold-network/solidity-contracts` so instead of replacing 0.8.9 with 0.8.17 we have them both. Hardhat should pick the right compiler: https://hardhat.org/hardhat-runner/docs/advanced/multiple-solidity-versions#multiple-solidity-versions